src/python/ensembl/io/genomio/seq_region/dump.py

```diff
@@ -192,7 +195,7 @@ def get_attribs(seq_region: SeqRegion) -> List:
         seq_region (SeqRegion): The seq_region from which the attribs are extracted.
 
     Returns:
-        List: All attribs as a dict with 'value' and 'source' keys.
+        List: All attribs as a list of dicts with 'value' and 'source' keys.
```

Comment:
```suggestion
        All attributes as a list of dictionaries with 'value' and 'source' keys.
```
The return type is already included in the documentation from the type hints.